airbyte-integrations/connectors/source-postgres/src/test/resources/expected_spec.json

```diff
@@ -133,23 +133,23 @@
               },
               "client_certificate": {
                 "type": "string",
-                "title": "Client Certificate (Optional)",
+                "title": "Client Certificate",
```

Comment:
The changes to this file were required for PR https://github.com/airbytehq/airbyte/pull/17544